src/App.js

```diff
@@ -1,26 +1,43 @@
-import React from 'react';
-import logo from './logo.svg';
+import React, { Component } from 'react';
+import request from 'superagent';
 import './App.css';
+import QuoteList from './QuoteList.js';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
-}
+export default class App extends Component {
 
-export default App;
+  state = {
+    searchQuery: null,
+    displayOrder: 'asc',
+    pokemonName: '',
+    pokemonType: '',
+    pokemonAttack: '',
+    pokemon: [],
+  }
+
+// checks if input data has been change and updates value
+  handleChange = (e) => {
+    const value = e.target.value;
+    this.setState({ searchQuery: value });
+  }
+
+  handleClick = async () => {
+    const fetchedData = await request.get(`https://alchemy-pokedex.herokuapp.com/api/pokedex?pokemon=${this.state.searchQuery}`)
+    this.setState ({ pokemon: fetchedData.body.results });
+    console.log(fetchedData);
+    console.log(fetchedData.body);
+    
+  }
+
+
+  render() {
+    return(
+      <main>
+        <input onChange={this.handleChange} name="searchQuery" />
+        <button onClick={this.handleClick} />
```

Comment:
if you make this a regular opening and closing tag you can put something like "search" inside the tags to make the button easier to press